reporters/allure-testinfo-adapter.js

```
@@ -0,0 +1,27 @@
+// reporters/allure-testinfo-adapter.js
+const { allure } = require('allure-playwright');
```

This file imports a module-level Allure instance from `allure-playwright`. This creates a singleton that you're delegating metadata calls to, which violates the required behavior. Specifically, the project corrections state: **"HIGH: Remove any fallback from testInfo.allure to a module-level singleton for attaching Allure metadata (e.g., code like const a = testInfo?.allure ?? singleton). All metadata operations must use the Playwright adapter bound to the current test."**

Do not use the module-level `allure` here to attach labels for the current test. Instead, rely on `testInfo.allure` provided by Playwright (and fail fast if it's absent).